### ISSUE TITLE
Share IPA

### ISSUE THREAD

**Issue by AstudilloMarillo** [Lukas1h/BeReal-iOS#2]
Hi, can you please share an IPA of this app? I'd love to try it out.

**Comment by Lukas1h** [comment]
Yeah, I'll try to upload an IPA as a release in the next few days. Honsestly, I'm not sure if it will even work, since BeReal has changed there API quite a bit in the last few months.

In the mean time, you should check out [Bea](https://github.com/yandevelop/Bea), a tweek for BeReal that pretty much does everything this app does.

**Comment by AstudilloMarillo** [comment]
> In the mean time, you should check out [Bea](https://github.com/yandevelop/Bea), a tweek for BeReal that pretty much does everything this app does.

I just tried this. Unfortunately, sideloading doesn't work for Bea atm. Only with jailbreak it works, but I don't have my iPhone jailbroken. Hope yours will work, but it might be wishful thinking after reading through the login issues on Bea.

**Comment by Lukas1h** [comment]
There's also [BeFake](https://github.com/FH-DEV1/BeFake), which isn't a native iOS app unfortunately, but feels very similar to the actual BeReal app when installed as a PWA.
